packages/website/src/features/Router/RouterPage.tsx

```diff
@@ -0,0 +1,220 @@
+'use client';
+import {
+  useTheme,
+  Flex,
+  Box,
+  Heading,
+  Text,
+  Link,
+  Code,
+} from '@chakra-ui/react';
+import NextLink from 'next/link';
+import React from 'react';
+import { CodePreview } from '@/components/CodePreview';
+import { CommandPreview } from '@/components/CommandPreview';
+
+const code1 = `name = "sample-router-project"
+version = "0.1"
+description = "Sample Router Project"
+
+[contract.Counter]
+artifact = "Counter"
+
+[contract.AnotherCounter]
+artifact = "AnotherCounter"
+
+[router.Router]
+contracts = [
+  "Counter",
+  "AnotherCounter",
+]
+depends = [
+  "contract.Counter",
+  "contract.AnotherCounter",
+]`;
+
+const code2 = `[setting.admin]
+defaultValue = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
+
+[provision.proxy]
+source = "transparent-upgradable-proxy:4.9.3"
+options.admin = "<%= settings.admin %>"
+options.implementation = "<%= contracts.Router.address %>"
+options.abi = "<%= contracts.Router.abi %>"
+depends = ["router.Router"]`;
+
+export const RouterPage = () => {
+  const theme = useTheme();
+
+  return (
+    <Flex
+      background={theme.gradients.dark}
+      backgroundAttachment="fixed"
+      flex="1"
+      py={[0, 0, 8]}
+    >
+      <Box
+        p={8}
+        maxW="container.md"
+        m="auto"
+        background="black"
+        border="1px solid"
+        borderColor="gray.700"
+        borderRadius="md"
+      >
+        <Heading
+          as="h2"
+          size="lg"
+          fontWeight={600}
+          pb={4}
+          mb={4}
+          borderBottom="1px solid"
+          borderBottomColor="gray.800"
+          letterSpacing="0.2px"
+        >
+          Use a Synthetix Router
+        </Heading>
+        <Text mb={4}>
+          There is a limit to the size of smart contracts deployed on EVM
+          blockchains. This can create complications during the development of
+          protocols, where engineers may want an arbitrary amount of code to be
+          executable at a single address.
```

Comment:
additionally, its difficult to maintain large code files/inheritence trees, which single contract-based development leads to